src/validation/loader.rb

```diff
@@ -41,18 +41,14 @@ def question_tree
           Validation::Configure.new(combined_configure_data).tree
       end
 
-      # Returns a tree
+      # TODO: Remove this method
       def configure_section(section)
-        question_tree.children.find { |c| c.name == section }
+        question_tree.section_tree(section)
       end
 
-      # Returns a hash of identifiers and questions
+      # TODO: Remove this method
```

Comment:
If you think we should remove these 2 methods can you just remove them, or document why you think they should be removed but haven't been yet if this isn't currently possible? Otherwise the codebase just becomes littered with cruft.

Reply:
Removed